src/Data/Neuroglia.Data.Expressions.JQ/JQExpressionEvaluator.cs

```diff
@@ -79,85 +77,55 @@ public virtual bool Supports(string language)
         /// <inheritdoc/>
         public virtual object? Evaluate(string expression, object data, Type expectedType, IDictionary<string, object>? args = null)
         {
+            if (string.IsNullOrWhiteSpace(expression))
```

Comment:
Thrown at line 85 too

Reply:
Because `expression` has been trimmed of `${}` is the meanwhile. Condition at L80 validates if expression is null or empty, at L85 it basically valid it's not `${ }`